docs/spec/distribution/02_state.md

```diff
@@ -30,15 +30,16 @@ type FeePool struct {
 }
 ```
 
-### Validator Distribution
+## Validator Distribution
 
 Validator distribution information for the relevant validator is updated each time:
- 1. delegation amount to a validator is updated, 
+
+ 1. delegation amount to a validator is updated,
  2. a validator successfully proposes a block and receives a reward,
- 3. any delegator withdraws from a validator, or 
+ 3. any delegator withdraws from a validator, or
  4. the validator withdraws it's commission.
 
- - ValidatorDistInfo:  `0x02 | ValOperatorAddr -> amino(validatorDistribution)`
+- ValidatorDistInfo:  `0x01 | ValOperatorAddr -> amino(validatorDistribution)`
```

Comment:
`ProposerKey` uses this prefix.

Reply:
Actually, there's no `ValidatorDistInfo` this also needs to get updated

Reply:
reverted